packages/app-accounts/src/AddressInfoMvp.tsx

```diff
@@ -48,28 +47,9 @@ interface Props extends BareProps {
 
 function renderBalances (props: Props, allAccounts: string[], t: (key: string) => string): React.ReactNode {
   const { address } = props;
-
   const { api } = useApi();
-
-    // Populate staking/CENNZ and spending/CPAY IDs from the connected chain
-    const stakingAssetId = useCall<AssetId>(api.query.genericAsset.stakingAssetId as any, []);
-    if (stakingAssetId) {
-      assetsRegistry.add(
-        stakingAssetId.toNumber().toString(),
-        STAKING_ASSET_NAME
-      );
-    }
-  
-    const spendingAssetId = useCall<AssetId>(api.query.genericAsset.spendingAssetId as any, []);
-    if (spendingAssetId) {
-      assetsRegistry.add(
-        spendingAssetId.toNumber().toString(),
-        SPENDING_ASSET_NAME
-      );
-    }
-
-  const cennzBalance = useCall<'Balance'>(api.query.genericAsset.freeBalance as any, [assetsRegistry.getStakingAssetId(), address]);
-  const cpayBalance = useCall<'Balance'>(api.query.genericAsset.freeBalance as any, [assetsRegistry.getSpendingAssetId(), address]);
+  const cennzBalance = useCall<'Balance'>(api.query.genericAsset.freeBalance, [new AssetRegistry().getStakingAssetId(), address]);
+  const cpayBalance = useCall<'Balance'>(api.query.genericAsset.freeBalance, [new AssetRegistry().getSpendingAssetId(), address]);
```

Comment:
is it possible to create just one instance of AssetRegistry and then call
 const assetReg = new AssetRegistry();
const cennzBalance = useCall<'Balance'>(api.query.genericAsset.freeBalance, [assetReg.getStakingAssetId(), address]);
   const cpayBalance = useCall<'Balance'>(api.query.genericAsset.freeBalance, [assetReg.getSpendingAssetId(), address]);

Reply:
AssetRegistry won't create a new instance as it's a singleton, I can change it too look like `AssetRegistry.instance()` instead